<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="2" passed="1" failed="1" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2025-10-29T14:37:56 IST" name="Default suite" finished-at="2025-10-29T14:41:38 IST" duration-ms="222086">
    <groups>
    </groups>
    <test started-at="2025-10-29T14:37:56 IST" name="Default test" finished-at="2025-10-29T14:41:38 IST" duration-ms="222086">
      <class name="Project.AdditionalExternalPOTests">
        <test-method is-config="true" signature="setupClass()[pri:0, instance:Project.AdditionalExternalPOTests@45752059]" started-at="2025-10-29T14:37:56 IST" name="setupClass" finished-at="2025-10-29T14:37:56 IST" duration-ms="6" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setupClass -->
        <test-method is-config="true" signature="setUp()[pri:0, instance:Project.AdditionalExternalPOTests@45752059]" started-at="2025-10-29T14:37:56 IST" name="setUp" finished-at="2025-10-29T14:38:30 IST" duration-ms="34435" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method signature="testTC004_ExternalPONumberValidation()[pri:1, instance:Project.AdditionalExternalPOTests@45752059]" started-at="2025-10-29T14:38:30 IST" name="testTC004_ExternalPONumberValidation" description="TC004: Verify validation error when External PO Number is empty" finished-at="2025-10-29T14:39:55 IST" duration-ms="85234" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testTC004_ExternalPONumberValidation -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:Project.AdditionalExternalPOTests@45752059]" started-at="2025-10-29T14:39:55 IST" name="tearDown" finished-at="2025-10-29T14:39:55 IST" duration-ms="153" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method is-config="true" signature="setUp()[pri:0, instance:Project.AdditionalExternalPOTests@45752059]" started-at="2025-10-29T14:39:55 IST" name="setUp" finished-at="2025-10-29T14:40:29 IST" duration-ms="33922" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method signature="testTC036_MultipleFileUpload()[pri:2, instance:Project.AdditionalExternalPOTests@45752059]" started-at="2025-10-29T14:40:29 IST" name="testTC036_MultipleFileUpload" description="TC036: Verify multiple files can be uploaded" finished-at="2025-10-29T14:41:37 IST" duration-ms="68143" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[TC036: FAILED - APPLICATION BUG DETECTED
Root Cause: File upload functionality is not working in the application
Evidence:
  - Manual upload attempt: FAILED (files not accepted)
  - Automated upload: Files sent but not displayed
  - Expected: Uploaded files should appear in the upload area
  - Actual: Upload area remains empty after file selection
Status: BUG REPORTED - Waiting for development fix]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: TC036: FAILED - APPLICATION BUG DETECTED
Root Cause: File upload functionality is not working in the application
Evidence:
  - Manual upload attempt: FAILED (files not accepted)
  - Automated upload: Files sent but not displayed
  - Expected: Uploaded files should appear in the upload area
  - Actual: Upload area remains empty after file selection
Status: BUG REPORTED - Waiting for development fix
at org.testng.Assert.fail(Assert.java:111)
at Project.AdditionalExternalPOTests.testTC036_MultipleFileUpload(AdditionalExternalPOTests.java:753)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:664)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:227)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:957)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:200)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
at org.testng.TestRunner.privateRun(TestRunner.java:848)
at org.testng.TestRunner.run(TestRunner.java:621)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:443)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:437)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:397)
at org.testng.SuiteRunner.run(SuiteRunner.java:336)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1280)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1200)
at org.testng.TestNG.runSuites(TestNG.java:1114)
at org.testng.TestNG.run(TestNG.java:1082)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testTC036_MultipleFileUpload -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:Project.AdditionalExternalPOTests@45752059]" started-at="2025-10-29T14:41:37 IST" name="tearDown" finished-at="2025-10-29T14:41:38 IST" duration-ms="168" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method is-config="true" signature="tearDownClass()[pri:0, instance:Project.AdditionalExternalPOTests@45752059]" started-at="2025-10-29T14:41:38 IST" name="tearDownClass" finished-at="2025-10-29T14:41:38 IST" duration-ms="1" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDownClass -->
      </class> <!-- Project.AdditionalExternalPOTests -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
